python/ray/tests/test_get_or_create_actor.py

```diff
@@ -0,0 +1,74 @@
+import sys
+import os
+import pytest
+
+import ray
+from ray._private.test_utils import (
+    run_string_as_driver_nonblocking,
+)
+
```

Comment:
should add some concurrent test to ensure no surprise?

Reply:
Yep, the verbose output one is actually a concurrency test. In the past, this pattern sometimes generated error logs under specific concurrent race conditions.